pyccel/codegen/printing/ccode.py

```diff
@@ -209,7 +210,45 @@ def _print_PyccelNot(self, expr):
         return '!{}'.format(a)
 
     def _print_Import(self, expr):
-        return '#include "{0}"'.format(expr.source)
+        return '#include <{0}>'.format(expr.source)
+
+    def _print_String(self, expr):
+        return "\"{}\"".format(expr.arg)
+
+    def _print_Print(self, expr):
+        self._additional_imports.add("stdio.h")
+        args = []
+        args_format = []
+        end = '\\n'
+        sep = ' '
+        for f in expr.expr:
+            if isinstance(f, ValuedVariable):
+                if f.name == 'sep'      :   sep = str(f.value)
+                elif f.name == 'end'    :   end = str(f.value)
+            elif f.dtype is NativeBool():
+                args_format.append("%s")
+                args.append('({}) ? "True" : "False"'.format(self._print(f)))
```

Comment:
Ideally it would be better to use `IfTernaryOperator` (the print method needs to be implemented)

Reply:
done.

Reply:
Personally I think this is dangerous. It may break any code using the words `True` or `False`, e.g:
```
True_bool = True
```
in c:
```
True_bool = True;
```
After preprocessing:
```
1_bool = 1;
```
I am against adding even more restrictions to the naming which is already difficult (see issue #227 )

Reply:
> but we can't use it here, it has no meaning to construct an IfTernaryOperator from a value or a condition.

I'm not entirely sure what you mean but looking at the code it looks like `IfTernaryOperator` is implemented in a strange way, probably because fortran does not have this facility. Does your implementation break the usage of `IfTernaryOperator`? I'm not sure where it is tested for fortran. For c it is used when collecting boolean arguments (see ast/cwrapper.py:L223)

Reply:
P.S. If you edit your comment instead of adding a new comment I do not receive an alert and am therefore unlikely to reply. Edit should be used for typos

Reply:
> probably because fortran does not have this facility

This is not true, it can be implemented in fortran using `merge`. I would therefore be in favour of modifying `IfTernaryOperator` so it no longer inherits from `If` and keeps its original form after `_visit_Assign` in parser/semantic.py. However this is completely out of the scope of this pull request. I will create an issue for it.

Given that fact, I am happy with your original code

Reply:
> Does your implementation break the usage of `IfTernaryOperator` ?

If so then your implementation of `_print_IfTernaryOperator` should be removed from this PR. You could move it to a new branch which can then be used by whoever wants to fix the `IfTernaryOperator`

Reply:
`True_bool = True;` will be `True_bool = 1;` not `1_bool = 1;.`
defines don't process subwords.
to see that try to compile your example by `gcc -E code.c`. This will output the preprocessed source code.

Reply:
okay. I will remove it.

Reply:
> defines don't process subwords.

You are right, sorry. In that case I would be interested to know what @yguclu and @saidctb think about using macros for this as `True` and `False` will be reserved anyway

Reply:
I agree to use  macros in this case because it will make the code more readable